### PR TITLE
Avseenkov.v.home work2 ex3

### PR DESCRIPTION
**Pull request by a-vov4eg**

### CODE REVIEW

File: homework/Vladimir Avseenkov.ex3_NumberService/Service/NumberService.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package Service;
+
+public class NumberService implements Runnable {
+    int startOfRange, endOfRange;
+    int totalNumbers = 0;
+    int totalEvenNumbers = 0;
+    int summary = 0;
+    String output = "";
```
Comment (javaGuruBY) [inline homework/Vladimir Avseenkov.ex3_NumberService/Service/NumberService.java:8]: outputOdd?

**Review by javaGuruBY — APPROVED** [review]
nice, add some tests meybe?

**Review by vladis1350 — APPROVED** [review]
Lfgm

**Review by TLittleProgrammer — APPROVED** [review]
lgfm

**Pull request merged**